app/models/ticket_subscription.rb

```diff
@@ -2,6 +2,15 @@ class TicketSubscription < ApplicationRecord
   belongs_to :ticket
   belongs_to :user
 
+  validates :ticket_id,
+    presence: true
+
+  validates :user_id,
+    presence: true,
+    uniqueness: {
```

Comment:
I do not oppose adding things.
If you purchase multiple tickets, purchase history may be required separately in preparation for additional purchase.